src/auth/FormElement.ts

```diff
@@ -333,9 +349,6 @@ export default class GmfAuthForm extends GmfBaseElement {
    * @private
    */
   checkUserMustChangeItsPassword_(): void {
```

Comment:
Shouldn't be named as `handleUserMustChangeItsPassword_`?

Reply:
I will rename it, it does the check outside so the name is not accurate anymore.